docker/docker_image_src.go

```diff
@@ -65,11 +73,37 @@ func (s *dockerImageSource) GetManifest() (manifest types.ImageManifest, unverif
 	if res.StatusCode != http.StatusOK {
 		return nil, "", errFetchManifest{res.StatusCode, manblob}
 	}
+	// TODO(remove, already set in manifest, below): Miloslav?
```

Comment:
I haven't well understood that the digest we get could not be the *right* one so in the past I embedded it to display it to the user - however my current understanding is that this digest is not to be trusted so I don't think it's needed to be exposed in skopeo - correct?